content/en/infrastructure/livecontainers/legacy.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Live Containers Legacy Configuration
+kind: documentation
+further_reading:
+- link: "/infrastructure/livecontainers/configuration"
+  tag: "Documentation"
+  text: "Configure live containers"
+---
+
+## Overview
+
+This page provides instructions for setting up Live Containers for older Agent versions. These instructions apply to Datadog Agent versions [7.21.1][1] through 7.27.0 and Cluster Agent [1.9.0][2] through 1.11.0.
```

Comment:
It looks like these links currently lead to tagging documentation, is that expected?